app/controllers/reports_controller.rb

```diff
@@ -1,12 +1,13 @@
 class ReportsController < ApplicationController
 
   def index
+    @report_builder = ReportBuilder.new(date: params[:date], cohort: params[:cohort])
   end
 
   def lengths_for_measurement
     @processed_file_id = params[:processed_file_id]
-    @total_animal_lengths = TaggedAnimalAssessment.lengths_for_measurement(@processed_file_id) +
-        UntaggedAnimalAssessment.lengths_for_measurement(@processed_file_id)
+    @total_animal_lengths = TaggedAnimalAssessment.lengths_for_measurement(params[:shl_case_number], params[:date])
+    # + UntaggedAnimalAssessment.lengths_for_measurement(@processed_file_id)
```

Comment:
Do you think this line will ever come back into play?

Reply:
So that's one of the areas I'm not sure about. That comment is originally from Ellen's PR that I merged into here. 

There's a couple other places, mostly in the ReportBuilder class, where I'm only looking up TaggedAnimalAssessments, but I'm not sure if the Growth Report is supposed to be for UntaggedAnimalAssessments as well. I'll circle back on all of that code after #117 is merged and I'm able to see exactly how the reports work.